flashinfer/gemm/gemm_base.py

```
@@ -3755,3 +3794,409 @@ def fp8_blockscale_gemm_sm90(
+        "mxfp8_gemm",  # TODO: check if this is correct
+        runners,
+        _FP8_GEMM_SM100_TUNING_CONFIG,  # TODO: check if this is correct
+        inputs,
+    )
+
+    runner(inputs=inputs, tactic=tactic)
+
+
+@supported_compute_capability([100, 103])
+def _cudnn_bmm_mxfp8_requirement(
+    A: torch.Tensor,
```

Ruff (0.14.10)

Unused function argument: `A`

(ARG001)